data/bedrock/latest/types.yml

```diff
@@ -2436,3 +2471,31 @@ FullContainerName:
 MovementEffectType: varint =>
    -1: invalid
    0: GLIDE_BOOST
+
+CameraAimAssistAction: u8 =>
+   # Sets aim-assist to use the settings from a CameraPresets aim_assist field.
+   0: set_from_camera_preset
+   # Clears aim-assist settings.
+   1: clear
+
+EntityBoundingBoxComponent:
+   scale: lf32
+   width: lf32
+   height: lf32
+
+MovementAttributesComponent:
+   movement_speed: lf32
+   underwater_movement_speed: lf32
+   lava_movement_speed: lf32
+   jump_strength: lf32
+   health: lf32
+   hunger: lf32
+
+CreativeItemCategory: li32 =>
```

Comment:
Avoid adding types here unless it's something that's reused in multiple places or really big